src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -64,7 +64,7 @@ public TermsOfUseHelper(TermsOfUseHelperParams touParams)
         }
 
 
-        public void Execute(bool preventReentrant = true)
+        internal void Execute(bool preventReentrant = true)
```

Comment:
Who are the clients of TermsOfUseHelper class? If DynamoRevit or DynamoStudio don't call it then we might not need to fix this. In that case we can make this class internal.